__tests__/seed.test.js

```diff
@@ -52,3 +52,46 @@ describe('GET /api', () => {
     })
 });
 
+describe('/api/articles/:article_id/comments', () => {
+    test("200: returns an array of comments for the given article_id", () => {
+        return request(app).get('/api/articles/3/comments')
+            .expect(200)
+            .then(({ body }) => {
+                expect(body.rows.length).toBe(2);
+                expect(body.rows[0]).toMatchObject(
```

Comment:
This is a little redundant here, we're checking for all comments. I'd love to see in your for each assert that each comment has the correct article id.

Reply:
I second this, rather than checking rows[0 a rows forEach would be better.